docs/docs/dev_docs/wallets/writing_an_account_contract.md

```diff
@@ -1,3 +1,96 @@
 # Writing an Account Contract
 
-Please use the [TUTORIAL-TEMPLATE](../../TUTORIAL_TEMPLATE.md) for standalone guides / tutorials.
\ No newline at end of file
+This tutorial will take you through the process of writing your own account contract in Noir, along with the Typescript glue code required for using it within a [wallet](./main.md).
+
+Writing your own account contract allows you to define the rules by which user transactions are authorised and paid for, as well as how their keys are managed and can be potentially rotated or recovered. In other words, lets you make the most out of [account abstraction](../../concepts/foundation/accounts/main.md#what-is-account-abstraction) in the Aztec network.
```

Comment:
Accepted, but I'm curious if we have a stance regarding the use of singular-they.

Reply:
>  I'm curious if we have a stance regarding the use of singular-they.

A stance? I've been using "they" as a singular since before I was born. :)

The reason I suggested replacing `their` with `user` is because the subject of your sentence is "user transactions" (not "users"), and so writing "their keys" implied the keys belonged to the "user transactions". I.e. "their" expands to "user transactions' keys", which doesn't make sense.

Reply:
(At least, that's how my brain was reading it)